stacker/session_cache.py

```diff
@@ -23,6 +25,9 @@ def get_session(region, profile=None):
         :class:`boto3.session.Session`: A boto3 session with
             credential caching
     """
+    if profile is None:
+        profile = default_profile
+
```

Comment:
Do you think it's worth adding a debug log in here so we can see what profiles/regions sessions are being created in?